exporters/tokens/src/helpers/colorHelper.ts

```diff
@@ -73,3 +75,21 @@ export const transformColorsToVariables = (
 
   return transformedValue;
 };
+
+export const filterColorCollections = (tokens: Token[]) => {
+  const COLLECTION_NAME = 'Theme tokens';
+
+  return tokens.filter((item) => {
+    // Include all tokens that are not of tokenType "Color"
+    if (item.tokenType !== TokenType.color) return true;
```

Comment:
**suggestion (if-minor):** Can you please use a block of `{...}` for better readibility?